src/View/AbstractTemplateView.php

```diff
@@ -377,7 +377,7 @@ protected function getCurrentParsedTemplate()
     {
         $currentRendering = end($this->renderingStack);
         $renderingContext = $this->getCurrentRenderingContext();
-        $parsedTemplate = $currentRendering['parsedTemplate'] ? $currentRendering['parsedTemplate'] : $renderingContext->getTemplateCompiler()->getCurrentlyProcessingState();
+        $parsedTemplate = isset($currentRendering['parsedTemplate']) ? $currentRendering['parsedTemplate'] : $renderingContext->getTemplateCompiler()->getCurrentlyProcessingState();
```

Comment:
This should probably be `!empty()` instead to be closer to the previous behavior.